spec/jobs/google_scraping_job_spec.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe GoogleScrapingJob, type: :job do
+  include ActiveJob::TestHelper
+  ActiveJob::Base.queue_adapter = :test
+
+  describe '#perform' do
+    context 'given valid keyword (with top position adwords)' do
```

Comment:
do we need to wrap in a context as we have only 1 test case here

Reply:
It is better to have context IMO because I think it will be more clearer when read the test.

Reply:
Yeah but I think we don't need the context in this case.
And also add more test to this spec in cb26838fe5557771358c2259c4376959339ef35c